doc/api/v8.md

```diff
@@ -203,6 +204,8 @@ Writes out a header, which includes the serialization format version.
 
 #### serializer.writeValue(value)
 
+* `value` {Object}
```

Comment:
Ditto (not full `{any}` (no symbols, for example), but a comment below warns about this, so we can state `{any}` to include primitives).

Reply:
Maybe we should also add a comment right here?

Reply:
@BridgeAR: please explain?

Reply:
I was just wondering about adding the same warning that describes the details here as well when changing it to `{any}`. Or we could just move that part.